tests/execute/result/special/special-chars.fmf

```diff
@@ -0,0 +1,9 @@
+execute:
+    how: tmt
+discover:
+    how: shell
+    tests:
+      - name: '0..7 special: characters: *$@|&>< in: the: name'
+        test: "tmt-report-result '0..7 \"special\": \" characters: *$@|&>< in: the: name' PASS"
```

Comment:
I think `result: restraint` needs to be set as well to actually parse the generated results file.